torch/optim/optimizer.py

```diff
@@ -270,9 +270,10 @@ def _to_scalar(x):
               implementation, pass False for either foreach or fused. """
 
 _capturable_doc = r"""capturable (bool, optional): whether this instance is safe to
-            capture in a CUDA graph. Passing True can impair ungraphed performance,
-            so if you don't intend to graph capture this instance, leave it False
-            (default: False)"""
+            capture in a graph, whether for CUDA graphs or for `torch.compile` support.
+            Tensors are only capturable when on supported accelerators like CUDA, XPU,
```

Comment:
nit: you can link to the accelerators page if you want to make sure the list stays up to date via ```:ref:`accelerator<accelerators>` ```